### PR TITLE
Tests for Elytron key and credential store configuration

### PR DESCRIPTION
**Pull request by zebrik**

### CODE REVIEW

**Comment by zebrik** [comment]
@rhatlapa please review but not merge yet - waiting for upcoming HAL upgrade

File: common/src/main/java/org/jboss/hal/testsuite/page/config/elytron/AbstractElytronConfigPage.java
Comment (rhatlapa) [inline common/src/main/java/org/jboss/hal/testsuite/page/config/elytron/AbstractElytronConfigPage.java]: JavaDoc would be nice

File: common/src/main/java/org/jboss/hal/testsuite/page/config/elytron/AbstractElytronConfigPage.java
Comment (rhatlapa) [inline common/src/main/java/org/jboss/hal/testsuite/page/config/elytron/AbstractElytronConfigPage.java:41]: If this part is general, it might be worth extracting it to some general page which contains config area, just a suggestion.

**Review by rhatlapa — APPROVED** [review]
Just one note regarding missing javadoc for abstract class, otherwise LGTM

File: common/src/main/java/org/jboss/hal/testsuite/page/config/elytron/AbstractElytronConfigPage.java
Comment (zebrik) [reply common/src/main/java/org/jboss/hal/testsuite/page/config/elytron/AbstractElytronConfigPage.java:41]: It might me worth to move this to the ConfigPage. @honza-kasik  WDYT? May be in another PR.

**Comment by zebrik** [comment]
@rhatlapa JavaDoc added.

**Comment by zebrik** [comment]
@honza-kasik look at the changes in the `ConfigChecker`. You might find them useful ;-)

**Pull request merged**
